src/types/date-time-picker.ts

```diff
@@ -1,8 +1,26 @@
 import type { AnyElementOf } from '@/types/utils'
 
+// v-calendar defines these types internally, but does not export them
+export interface SimpleDateParts {
+  year: number
+  month: number
+  day: number
+  hours: number
+  minutes: number
+  seconds: number
+  milliseconds: number
+}
+export type DateSource = Date | string | number
+export type DatePickerDate = DateSource | Partial<SimpleDateParts> | null
+export type DatePickerRangeObject = {
+  start: Exclude<DatePickerDate, null>
+  end: Exclude<DatePickerDate, null>
+}
+export type DatePickerModel = DatePickerDate | DatePickerRangeObject
```

Comment:
As far as I can tell, there's no importing types from v-calendar.

I also tried `yarn add @types/v-calendar` as suggested in a random Stack Overflow post, that is not a thing.